client/app/scripts/liveblog-edit/posts.service.js

```diff
@@ -157,7 +180,8 @@ define([
                             blog: blog_id,
                             text: item.text,
                             meta: item.meta,
-                            item_type: item.item_type
+                            item_type: item.item_type,
+                            name: item.meta && item.meta.name
```

Comment:
`name` can be equals to `false` here. Is it possible in the model point of view ? It seems unnatural :)